exec/jdbc-all/pom.xml

```diff
@@ -874,6 +874,42 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>hadoop-2</id>
+      <build>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-enforcer-plugin</artifactId>
+          <executions>
+            <execution>
+              <id>enforce-jdbc-jar-compactness</id>
+              <goals>
+                <goal>enforce</goal>
+              </goals>
+              <phase>verify</phase>
+              <configuration>
+                <rules>
+                  <requireFilesSize>
+                    <message>
+                      The file drill-jdbc-all-${project.version}.jar is outside the expected size range.
+                      This is likely due to you adding new dependencies to a java-exec and not updating the excludes in this module. This is important as it minimizes the size of the dependency of Drill application users.
+                    </message>
+                    <maxsize>47600000</maxsize>
```

Comment:
If plugin configs have only different `maxsize`, you can define it as a property and set that property value in the profile instead of providing plugin configs.

Reply:
Sounds like something that we should make a property anyway.

Reply:
Could we add something to the developer docs as well?

Reply:
Agree with all. Property is added and `## drill-jdbc-all JAR maxsize` doc header is added to `docs/dev/Maven.md`